app/views/users/profile.html.erb

```diff
@@ -213,6 +215,9 @@
 </div>
 
 <script type="text/javascript">
+  $('#btn-token').click(function() {
+    alert("Your access token is: <%= @user.user.token %>");
```

Comment:
It can still be selected. However, we could later add helpful "copy to clipboard" shortcuts.